Snoop.Core/Controls/ValueEditors/EditorTemplates.xaml

```diff
@@ -507,13 +501,11 @@
                        Foreground="{DynamicResource Snoop.Brushes.Default.Foreground}"
                        Text="{Binding ColorText}" />
             <Border Grid.Column="2"
```

Comment:
Are you sure the changes here are required?
Those values are obtained from BrushStop, which is a custom class where Color is not nullable.
Do you have a sample/repro where the error you mentioned is logged?

Reply:
Ah, it's not required but because your new converter freezes the created brush we save resources.

Reply:
Only the Border.Background (line 515) raised those errors (and needs to be changed) but I think the current Rectangle.Fill (lines 283, 308) can also result in binding errors.

To reproduce the error you can use any WPF application. I have created a blank project just now to test it and the debug output is like this https://gist.github.com/Garzuuhl/4850af94f8f99092b95c3b57bae4b05d

Reply:
They do result in binding errors if the color property is nullable and it's value is really null.
Good catch.